src/routes/(site)/s23positions/position-list.svelte

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+  import Position from './position.svelte';
+  import type { ClubPosition } from './position';
+  import type { TOOLS } from './data';
+
+  export let data: ClubPosition<keyof typeof TOOLS>[];
+</script>
+
+{#each data as position}
+  <section class="position">
+    <Position data={position} />
+  </section>
```

Comment:
Nit: What is the purpose of `<section class="position" />`?

Reply:
It seems to be a remnant from a previous HTML layout. Removed by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/754/commits/8a4be86b99c6f2e25329d4e7b720dedb5845ce06>.